src/org/exist/xquery/functions/util/LogFunction.java

```diff
@@ -173,10 +174,7 @@ public Sequence eval(Sequence[] args, Sequence contextSequence) throws XPathExce
                 final String logname = args[1].getStringValue();
 
                 // Use specific logger when provided
-                Logger logger = LOG;
-                if (logname != null && logname.length() > 0) {
-                    logger = LogManager.getLogger(logname);
-                }
+                Logger logger = StringUtils.isBlank(logname) ? LOG : LogManager.getLogger(logname);
```

Comment:
it is essentially the same right? I use this class basically always..... for readability.
The isBlank() also checks for whitespace characters....